PeerioTechnologies/peerio-desktop
# fix: LegalDialog can open privacy/terms within popup

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/13211/desktop-make-privacy-and-terms-of-use-links-always-open-as-popup-in-app-instead-of-external-link

#### Testing instructions  
* when LegalDialog ("terms of use" dialog or "privacy" dialog) is open, clicking on "terms of use" or "privacy policy" links no longer opens an external link, instead switches to the other text inside the same popup
* telemetry for these events has also been updated
* on the code side LegalDialog handles duration telemetry itself, simplifying the TermsOfUse.tsx file

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
